toolkit/util/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+plugins {
```

Comment:
We like self-descriptive code. Could we rename "utils" to something more specific? Who's the utils client?

Reply:
Well, this module is responsible for spec serialization. Also, I'd like to move here a specific logic, that is used  for example in Kotlin Notebook - applying the flavour for spec if it is not set (this is necessary for auto-applying the environment theme). 
So generally, this is useful stuff for those who create a Lets-plot spec on the client and send it (like Kotlin Notebook).

Reply:
The package is `package org.jetbrains.letsPlot.toolkit.jupyter`.

If this project is jupyter specific I would propose to rename it to "jupyter-util".

On the other hand, if it's not jupyter specific then we better remove "jupyter" from the package name and rename the project to smth like "kotlinx-serialization" (and the package - accordingly).

Reply:
Ok, may be just "serialization"?

Reply:
I'm not sure now. This is generic serialization code not related to LP or Jupyter. Strictly speaking it shouldn't be here at all. Do you use it in Jupyter integration?

Reply:
Yes, integration uses serialization of plot spec to pass it to Notebook. Notebook uses deserialisation part of this code to get the spec (and then use it in for Swing rendering).

Reply:
As I said above, this can be very useful for users who want to transfer plot spec (create it on the client and send it to the server, for example).

Reply:
Or, let's remove mentioning of letsPlot and spec:
```

typealias LetsPlotSpec = Map<String, Any>

fun serializeSpec(spec: LetsPlotSpec): JsonElement {
    return serialize(spec)
}
```

Next, this module is looking as a patch/missing feature for kotlinx.serilization.json wherefore we can name the package as "org.jetbrains.letsPlot.toolkit.kotlinx.json"  and the project name would be "kotlinx-json".
What do you think?

Reply:
But why would we remove the mention of lets-plot? We can more generally say that it is a `serialisation of a Map that contains nested only Map, List and primitives`, but it is exactly a serialisation of Lets-Plot spec. Yes, the module does not use Lets-plot directly, but this function is restricted to work specifically with Lets-Plot spec. If the spec structure is changed in the future (like adding a new possible type, or something like that), then we will also have to change this logic. So this implementation depends directly on Lets-Plot.

Reply:
But JSON is essentially just Maps, Lists and primitives. I don't see anything that is specific for lets-plot.

Reply:
Ok

Reply:
Renamed to `json`. All Lets-plot mentions are removed.